tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -184,40 +189,45 @@ export function useCIDefaults(projectName = 'test-project'): Promise<void> {
 
 export async function useCIChrome(projectName: string, projectDir = ''): Promise<void> {
   const protractorConf = path.join(projectDir, 'protractor.conf.js');
-  const chromePath = require('puppeteer').executablePath();
-
-  // Use Puppeteer in protractor if a config is found on the project.
   if (fs.existsSync(protractorConf)) {
-    const protractorPath = require.resolve('protractor');
-    const webdriverUpdatePath = require.resolve('webdriver-manager/selenium/update-config.json', {
-      paths: [protractorPath],
-    });
-    const webdriverUpdate = JSON.parse(await readFile(webdriverUpdatePath)) as {
-      chrome: { last: string };
-    };
-    const chromeDriverPath = webdriverUpdate.chrome.last;
-
+    // Ensure the headless sandboxed chrome is configured in the protractor config
     await replaceInFile(
       protractorConf,
       `browserName: 'chrome'`,
       `browserName: 'chrome',
       chromeOptions: {
-        args: ['--headless'],
-        binary: String.raw\`${chromePath}\`,
+        args: ['--headless', '--no-sandbox', '--disable-gpu', '--disable-dev-shm-usage'],
+        binary: String.raw\`${process.env.CHROME_BIN}\`,
       }`,
     );
     await replaceInFile(
       protractorConf,
       'directConnect: true,',
-      `directConnect: true, chromeDriver: String.raw\`${chromeDriverPath}\`,`,
+      `directConnect: true, chromeDriver: String.raw\`${process.env.CHROMEDRIVER_BIN}\`,`,
+    );
+  }
+
+  const karmaConf = path.join(projectDir, 'karma.conf.js');
+  if (fs.existsSync(karmaConf)) {
+    // Ensure the headless sandboxed chrome is configured in the karma config
+    await replaceInFile(
```

Comment:
This was required for the `generate/confg/type-karma.ts` test that creates a new karma config.